language/stdlib/modules/0L/LibraAccount.move

```diff
@@ -942,10 +942,14 @@ module LibraAccount {
         payer_sig: &signer,
         payee: address
     ) acquires LibraAccount, Balance, AccountOperationsCapability {
+        let bootstrap_amount = 1000000; // 1 gascoin, without scaling representation 
```

Comment:
Cant we have this as a const at the top?

Reply:
Yes good idea